app/models/google_calendar/event_editor.rb

```diff
@@ -0,0 +1,44 @@
+module GoogleCalendar
+  class EventEditor
+    def initialize(credentials)
+      @credentials = credentials
+      @calendar_service = GoogleCalendar::Client.new(credentials).calendar_service
+    end
+
+    def finish(conference_room_id, event_id)
+      @conference_room_email = ConferenceRoom.find(conference_room_id).email
```

Comment:
I don't think `@conference_room_email` and `@event` should be instance variables. They are not properties of the class, cause `EventEditior` allows to edit any events, not only these particular. After invoking `finish` they no longer make sense as a part of this class. You should pass them around as method arguments.